tests/test_couch/test_couch_parsers.py

```diff
@@ -7,6 +7,9 @@
 logging.basicConfig(level=logging.DEBUG)
 
 SIMPLE = '2015-10-31 18:32:03,963 [:mvp-pampaida] /a/mvp-pampaida/receiver/630916e49084b142c0a5a69c3a52b9b3/ PUT None d3abf611f2acdc7b4c32f7ebf4982a88 0:00:00.191515'
+URL_NO_TASK = '2015-10-31 18:32:03,963 [:mvp-pampaida] /a/mvp-pampaida/receiver/630916e49084b142c0a5a69c3a52b9b3/ - commcarehq PUT 200 None d3abf611f2acdc7b4c32f7ebf4982a88 0:00:00.191515'
+NO_URL_NO_TASK = '2015-10-31 18:32:03,963 [:mvp-pampaida] - - commcarehq PUT 200 None d3abf611f2acdc7b4c32f7ebf4982a88 0:00:00.191515'
+NO_URL_TASK = '2015-10-31 18:32:03,963 [:mvp-pampaida] - corehq.apps.tasks.build_app commcarehq PUT 200 None d3abf611f2acdc7b4c32f7ebf4982a88 0:00:00.191515'
```

Comment:
nit: an http method (like `PUT` here) wouldn't show up for a celery task, right? Not sure if that matters, but wanted to make sure the test is representative

Reply:
why not? this is the http method of the couch request, not the HQ request.

Reply:
Oh my, sorry I just woke up. Yeah that makes total sense.